core/controllers/admin.py

```diff
@@ -134,6 +136,14 @@ def post(self):
             elif self.payload.get('action') == 'reload_collection':
                 collection_id = self.payload.get('collection_id')
                 self._reload_collection(collection_id)
+            elif self.payload.get('action') == 'generate_dummy_explorations':
+                num_dummy_exps_to_generate = self.payload.get(
+                    'num_dummy_exps_to_generate')
+                num_dummy_exps_to_publish = self.payload.get(
+                    'num_dummy_exps_to_publish')
+                if num_dummy_exps_to_generate >= num_dummy_exps_to_publish:
```

Comment:
Better, but still not quite what I meant. Remember how, in the user bio PR, we checked for problems and then raised an exception, then did the "main thing"?

Here, your control flow should be:
- if var1 is not a number, raise 400 (InvalidInputError)
- if var2 is not a number, raise 400 (InvalidInputError)
- if var1 > var2, raise 400 (InvalidInputError)
- do the main processing